data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/stream/ShardConsumerFactory.java

```diff
@@ -84,10 +88,13 @@ public Runnable createConsumer(final StreamPartition streamPartition,
         String tableArn = getTableArn(streamPartition.getStreamArn());
         TableInfo tableInfo = getTableInfo(tableArn);
 
+        LOG.debug("Create shard consumer for {} with shardIter {}", streamPartition.getShardId(), shardIterator);
```

Comment:
I think it would be better to combine these log lines. Otherwise they will look unrelated when reviewing logs.

```
 LOG.debug("Create shard consumer for {} with shardIterator {} and lastShardIterator",
  streamPartition.getShardId(), shardIterator, lastShardIterator);
```

Reply:
I agree. Only a shard iterator is too long (almost 1000 characters), so it's better to keep them in different lines.

Reply:
That makes sense, so yes, let's keep it this way.